contentful_management/ui_extension.py

```diff
@@ -0,0 +1,79 @@
+from .resource import Resource
+
+
+"""
+contentful.ui_extension
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This module implements the UIExtension class.
+
+API Reference: https://www.contentful.com/developers/docs/references/content-management-api/#/reference/ui-extensions
+
+:copyright: (c) 2017 by Contentful GmbH.
+:license: MIT, see LICENSE for more details.
+"""
+
+
+class UIExtension(Resource):
+    """
+    API Reference: https://www.contentful.com/developers/docs/references/content-management-api/#/reference/ui-extensions
+    """
+
+    def __init__(self, item, **kwargs):
+        super(UIExtension, self).__init__(item, **kwargs)
+        self.extension = item.get('extension', {})
+
+    @property
+    def source(self):
+        return self.extension.get('src', '')
+
+    @source.setter
+    def source(self, value):
+        self.extension['src'] = value
+
+    @property
+    def name(self):
+        return self.extension.get('name', '')
+
+    @name.setter
+    def name(self, value):
+        self.extension['name'] = value
+
+    @property
+    def field_types(self):
+        return self.extension.get('fieldTypes', [])
+
+    @field_types.setter
+    def field_types(self, value):
+        self.extension['fieldTypes'] = value
+
+    @property
+    def sidebar(self):
+        return self.extension.get('sidebar', False)
+
+    @sidebar.setter
+    def sidebar(self, value):
+        self.extension['sidebar'] = value
```

Comment:
Hmm, I was just thinking, that I could add a non boolean type here, lets say an object and could potentially crash the parsing to json once it is about to go out to contentful.